authors/apps/authentication/utils.py

```diff
@@ -0,0 +1,33 @@
+import jwt
+import os
+from datetime import datetime, timedelta
+from django.conf import settings
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+
+# Method to Send emails
+def send_link(self, email):
```

Comment:
This is no longer a method, it's a function. So it doesn't need to take `self` as an argument as self is an instance of a Class